clients/drcachesim/common/trace_entry.h

```diff
@@ -395,7 +395,7 @@ typedef enum {
      * This marker is emitted prior to each system call when -enable_kernel_tracing is
```

Comment:
About the PR description: we need to define what is "streaming decoding". I thought it meant decoding multiple system calls with the same drpt2ir instance. But we decided to move that part to the next PR so we're still opening-closing PT events for each system call. So in this PR, we're **not** performing a streaming decode, but only a regular decode right? Please update the PR description which still mentions "enable the streaming decoding of PT data"

Reply:
Done.